src/OpenIddict.Core/OpenIddictModule.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Microsoft.AspNet.Builder;
+
+namespace OpenIddict {
+    /// <summary>
+    /// Defines an OpenIddict module.
+    /// </summary>
+    public class OpenIddictModule {
+        /// <summary>
+        /// Gets or sets the position of the module in the ASP.NET pipeline.
+        /// </summary>
+        public int Position { get; set; }
```

Comment:
This property is used for registering a module in the asp.net 5 pipeline, what's happening if two modules are registered using same position? since asp.net pipeline is based on registration order does we need to take care of this case?